.github/workflows/drupal.yml

```diff
@@ -0,0 +1,71 @@
+name: Drupal
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.3', '7.4']
+
+    services:
+      mysql:
+        image: druidfi/db:mysql5.7-drupal
+        ports:
+          - 3306/tcp
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
This can be sped up by using fetch-depth: 1 (like https://github.com/City-of-Helsinki/drupal-helfi/blob/dev/.github/workflows/ci.yml#L21)

Reply:
Thanks, updated